enterprise/models.py

```diff
@@ -4490,6 +4496,8 @@ def recent_action(self):
         """
         Return the timestamp of the most recent action relating to the membership
         """
+        if self.errored_at:
+            return self.errored_at
         if self.is_removed:
```

Comment:
If they are removed, don't we want to show that action before the errored at one? It seems like the more relevant one

Reply:
I think that it's possible the error could occur while we attempt to remove a learner so in that case there wouldn't be a removed status and we would want to display the error time.